internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/WriteComputeWithActiveActiveReplicationTest.java

```diff
@@ -1140,10 +1140,10 @@ public void testAAReplicationForPartialUpdateOnListField() throws IOException {
     verifyFLMRecord(storeName, dc0RouterUrl, key2, "", expectedArrayFieldVal, Collections.emptyMap());
     verifyFLMRecord(storeName, dc1RouterUrl, key2, "", expectedArrayFieldVal, Collections.emptyMap());
 
-    // The deleted element with the same timestamp should not be added back with PartialPut
+    // The deleted element with the same timestamp should be added back with PartialPut
     ubV1 = new UpdateBuilderImpl(wcSchemaV1);
     ubV1.setNewFieldValue(INT_ARRAY_FIELD, Arrays.asList(66, 99));
-    expectedArrayFieldVal = Arrays.asList(99);
+    expectedArrayFieldVal = Arrays.asList(66, 99);
```

Comment:
`66` was deleted with `ts = 2`, why adding it back is the correct behavior?

Reply:
Yes, after some thinking I think collection merge delete is should has less priority than PUT. 

Consider a collection merge delete (remove(Items/Keys)From(List/Map)) vs PUT has same TS = T0

For field level TS, it should be fine, but it the full PUT result in TS changing from field-level TS -> value-level TS, then the following collection merge delete won't be updated as it requires UPDATE operation TS > value-level TS record to proceed.

Reply:
We can also make value-level TS accept UPDATE == current TS, but it will make logic complicated as we might expand it and then collapse it back if we found that the UPDATE has changed nothing.

Reply:
But I just realized that this setting does not treat PUT vs UPDATE.setField on the same TS equally on the value-level TS. Let me think about this, but I don't think I should introduce much more change to this PR, but maybe another follow up PR along with the discussion happening in the write-path channel.